test/inductor/test_cpu_repro.py

```diff
@@ -1457,11 +1457,14 @@ def _internal_check(
         with config.patch({"cpp.fallback_scatter_reduce_sum": True}):
             _internal_check(fn, inps, "aten.scatter_reduce_")
 
-        with set_num_threads(1):
-            _internal_check(fn, inps, _target_code_check_not="aten.scatter_reduce_")
-
-        with config.patch({"cpp.dynamic_threads": True}), set_num_threads(1):
-            _internal_check(fn, inps, "aten.scatter_reduce_")
+        if "ATen parallel backend: OpenMP" in torch.__config__.parallel_info():
```

Comment:
May be better to check `torch.has_openmp`

Reply:
Thanks for the suggestions and changing to use `torch.has_openmp` is clearer.
Let's check if it can fix the issue. Since from the code, https://github.com/pytorch/pytorch/blob/b8307513e57f8beaf99daff342a23d705a417e11/caffe2/CMakeLists.txt#L16-L49, seems when `INTERN_BUILD_MOBILE` turns on, `AT_PARALLEL_NATIVE` and `USE_OPENMP` can be set to True at same time.